packages/app/src/components/swap/exit.tsx

```diff
@@ -175,7 +175,7 @@ const ExitPool: FC<Props> = ({
 
   return (
     <Box pad={{ horizontal: "large", top: "small" }}>
-      <Button primary onClick={handleShow} label="Exit Pool" />
+      <SecondaryButton secondary color="brand" onClick={handleShow} label="Exit Pool" />
```

Comment:
the namings here are confusing, applying `secondary` to a `SecondaryButton`. Related: we should not export a `Button` if we also use the underlying `Button` component elsewhere in the app.